views/index.html

```diff
@@ -161,7 +161,51 @@ <h2 class="about-header">Skills</h2>
                 </div>
             </article>
         </section>
+
     </main>
+    <footer class="footer" id="footer">
+        <img class="img-footer" src="../images/image_geometry_1.png" alt="footer image">
+        <section class="intro">
+            <h2 class="name">Get started</h2>
+            <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                incididunt ut labore et dolore magna.</p>
+                <form  action="https://formspree.io/f/mpzbqaoe" method="post">
+                    <ul class="info">
```

Comment:
- [ OPTIONAL]

- Please consider writing the `post` method all in capital. This should be `POST` instead of `post` as all HTTP requests are all in capital letters.